### PR TITLE
More agile cleanup

### PR DESCRIPTION
**Pull request by TessFerrandez**
- Simplified folder structure further + structure of lists (so they are not double nested)
- Fixed wording in some sections
- Cleaned up links

Minor changes to make the section more chill and easy to digest...

### CODE REVIEW

File: docs/agile-development/advanced-topics/collaboration/pair-programming-tools.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Effortless Pair Programming with GitHub Codespaces and VSCode
```
Comment (Doommius) [inline docs/agile-development/advanced-topics/collaboration/pair-programming-tools.md:1]:
```suggestion
# Effortless pair programming with GitHub Codespaces and VSCode
```

File: docs/agile-development/advanced-topics/collaboration/teaming-up.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Just as an example, agility debugging activities may include:
   - Are Acceptance Criteria enough and right?
   - Is everyone ready-to-go after taking the User Story/Task?
 
-- Running [Efficient Retrospectives](../../basics/ceremonies.md#retrospectives)
+- Running [Efficient Retrospectives](../../ceremonies.md#retrospectives)
```
Comment (Doommius) [inline docs/agile-development/advanced-topics/collaboration/teaming-up.md:53]:
```suggestion
- Running [efficient retrospectives](../../ceremonies.md#retrospectives)
```

File: docs/agile-development/ceremonies.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+- Estimating stories using story points in smaller project does not always provide the value it would in bigger ones.
+- Avoid converting story points or t-shirt sizes to days.
+
+#### Measure estimation accuracy
+
+- Collect data to monitor estimation accuracy and sprint completion over time to drive improvements.
+- Use the sprint goal to understand if the estimation was correct. If the sprint goal is met: does anything else matter?
+
+#### Scrum Practices
```
Comment (Doommius) [inline docs/agile-development/ceremonies.md:102]:
```suggestion
#### Scrum practices
```

File: docs/agile-development/ceremonies.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+- Schedule enough time to ensure that you can have the conversation you need to get the correct plan an action and improve how you work.
+- Bring in a neutral facilitator for project retros or retros that introspect after a difficult period.
+
+Use the following retrospectives techniques to address specific trends that might be emerging on an engagement
+
+### 5 whys
+
+If a team is confronting a problem and is unsure of the exact root cause, the 5 whys exercise taken from the business analysis sector can help get to the bottom of it. For example, if a team cannot get to *Done* each Sprint, that would go at the top of the whiteboard. The team then asks why that problem exists, writing that answer in the box below.  Next, the team asks why again, but this time in response to the *why* they just identified. Continue this process until the team identifies an actual root cause, which usually becomes apparent within five steps.
+
+### Processes, tools, individuals, interactions and the Definition of Done
```
Comment (Doommius) [inline docs/agile-development/ceremonies.md:144]: other places dod is all lower cases, should this be the case here as well ?

```suggestion
### Processes, tools, individuals, interactions and the definition of done
```

File: docs/design/design-reviews/recipes/engagement-process.md [hunk 5]
```diff
@@ -58,7 +58,7 @@ It is also a great time to check in on design reviews
 
 All design artifacts should be treated as a living document. As requirements change or uncover more unknowns the dev crew should retroactively update all design artifacts. Missing this critical step may cause the customer to incur future technical debt. Artifacts that are not up to date are `bugs` in the design.
 
-> **Tip:** Keep your artifacts up to date by adding it to your teams [Definition of Done](../../../agile-development/advanced-topics/team-agreements/definition-of-done.md) for all user stories.
+> **Tip:** Keep your artifacts up to date by adding it to your teams [Definition of Done](../../../agile-development/team-agreements/definition-of-done.md) for all user stories.
```
Comment (Doommius) [inline docs/design/design-reviews/recipes/engagement-process.md:61]:
```suggestion
> **Tip:** Keep your artifacts up to date by adding it to your teams [definition of done](../../../agile-development/team-agreements/definition-of-done.md) for all user stories.
```

**Review by Doommius — APPROVED** [review]
Looks good, Keep up the great work tess, 

The content and text looks look, there's a few casing things but there's a lot of inconsistency over the entire repo,

Do we know if there is a good linter that can handle auto capitalization rules for consistency ?

**Comment by TessFerrandez** [comment]
> Looks good, Keep up the great work tess,
> 
> The content and text looks look, there's a few casing things but there's a lot of inconsistency over the entire repo,
> 
> Do we know if there is a good linter that can handle auto capitalization rules for consistency ?

Thank you,

Not sure about the auto-capitalization TBH, and I suspect it is very varying in the pages... so I think (and I know its tedious) that we will just have to keep doing it manually

**Comment by TessFerrandez** [comment]
Merging with 1 approver per discussion with EF team

**Comment by TessFerrandez** [comment]
Merging with 1 approver based on previous discussion with EF team

**Pull request merged**